src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -246,6 +246,11 @@ enum IntrinsicId
 			AppDomain_CreateInstanceFrom,
 			AppDomain_CreateInstanceFromAndUnwrap,
 			Assembly_CreateInstance,
+			Assembly_Location,
```

Comment:
Should we also add AssemblyName_CodeBase, AssemblyName_EscapedCodeBase, Module_Name, and Module_FullyQualifiedName?